integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/synapseconfig/MaxOpenConnections/nhttp/axis2.xml

```diff
@@ -796,8 +791,8 @@
 
             <handler name="HTTPLocationBasedDispatcher"
                      class="org.apache.axis2.dispatchers.HTTPLocationBasedDispatcher"/>
-            <handler name="MultitenantDispatcher"
-                     class="org.wso2.carbon.tenant.dispatcher.MultitenantDispatcher"/>
+            <!--<handler name="MultitenantDispatcher"-->
+                     <!--class="org.wso2.carbon.tenant.dispatcher.MultitenantDispatcher"/>-->
```

Comment:
Shall remove this instead of commenting out?

Reply:
Fixed in: https://github.com/wso2/micro-integrator/pull/590